### PR TITLE
Document find_node()'s owner argument behaviour

### LINKED ISSUE #10595: find_node() does not return instanced scenes
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Godot 3.0 alpha 1, Windows 10

**Issue description:**
find_node() does not see nodes that are instanced scenes. Its very annoying because I would often create a bunch of nodes in the main scene, reference them using find_node() and then convert that branch into its own scene at which point the find_node() stops working even though the name of the node stays the same and its still in the main scene as previously.

**Steps to reproduce:**
Create a node in a scene and then convert it to a scene without changing the name. The instanced scene will be invisible to find_node()

### PR DESCRIPTION
**Pull request by yvie-k**
Resolves #10595

### CODE REVIEW

File: doc/classes/Node.xml [hunk 1]
```diff
@@ -176,6 +176,7 @@
 			</argument>
 			<description>
 				Finds a descendant of this node whose name matches [code]mask[/code] as in [method String.match] (i.e. case sensitive, but '*' matches zero or more characters and '?' matches any single character except '.'). Note that it does not match against the full path, just against individual node names.
+				If [code]owned[/code] is [code]true[/code], this method only finds nodes whose owner is this node. This is especially important for scenes instanced through script, because those scenes don't have an owner.
```
Comment (akien-mga) [inline doc/classes/Node.xml]: instanced -> instantiated

Comment (Dragoncraft89) [reply doc/classes/Node.xml]: Changed

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**